python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -794,7 +796,7 @@ def reset_index(
             self.index = cudf_series.index
             return None
 
-    def contains_properly(self, other, align=True):
+    def contains_properly(self, other, align=True, allpairs=False):
```

Comment:
Update the document for the new argument, the new `polygon_index` column in the result. And better, give an example.